btrfs/datadog_checks/btrfs/btrfs.py

```diff
@@ -11,9 +12,11 @@
 
 import psutil
 from six import iteritems
+from six.moves import xrange
```

Comment:
can we just use range instead?

Reply:
I think `xrange` is valid here since it returns a generator instead of the full list. One part of the check that uses it is https://github.com/DataDog/integrations-core/blob/576925cfcbb898b5afeee36d97ff49e531421ddc/btrfs/datadog_checks/btrfs/btrfs.py#L148 which loops through all found devices.

Reply:
okay!

Reply:
This actually should have been in my PR description, so I added it there!